seqr/utils/elasticsearch/es_search.py

```diff
@@ -82,6 +82,20 @@ def __init__(self, families, previous_search_results=None, inheritance_search=No
         self._no_sample_filters = False
         self._any_affected_sample_filters = False
 
+    @staticmethod
+    def _parse_xstop(result):
+        xstop = result.pop(XSTOP_FIELD, None)
+        if xstop:
+            end_chrom, end = get_chrom_pos(xstop)
+            if end_chrom != result['chrom'] or end != result['end']:
+                if result['svType'] == 'INS':
```

Comment:
I talked to Alba about this one. It turns out for insertions they use the `CHR2` field to tell you where the sv came from, but we don't want to display the source as the position the way we do with translocations/ complex SVs. I think its out of scope to change this in the pipeline so this seemed like a reasonable workaround for now, but I am open to discussion

Reply:
I agree that it is a little complex to add it here, but it needs to display in the UI hover where I have added it and not as a complex interval